python/foglamp/services/core/api/support.py

```diff
@@ -20,6 +19,9 @@
 
 _FOGLAMP_DATA = os.getenv("FOGLAMP_DATA", default=None)
 _FOGLAMP_ROOT = os.getenv("FOGLAMP_ROOT", default='/usr/local/foglamp')
+_SYSLOG_FILE = '/var/log/syslog'
+__DEFAULT_LIMIT = 20
+__DEFAULT_OFFSET = 0
```

Comment:
no offset feature? however we defined this const

Reply:
removed